gate-core/src/main/java/com/netflix/spinnaker/gate/services/TaskService.java

```diff
@@ -22,14 +22,15 @@
 import java.util.*;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.stereotype.Service;
 
 @Service
+@ConfigurationProperties(prefix = "taskService")
```

Comment:
Sorry to keep hammering on this.  The typical pattern is to have a separate `@ConfigurationProperties` class that holds only the properties, and then to autowire that class here.

Reply:
Sure sec ;)